Rubberduck.Core/AutoComplete/AutoCompleteBase.cs

```diff
@@ -14,23 +18,37 @@ protected AutoCompleteBase(string inputToken, string outputToken)
         public string InputToken { get; }
         public string OutputToken { get; }
 
+        private readonly Stack<Tuple<int, string>> OriginalLines;
+
+        public void Revert()
+        {
+            if (OriginalLines.Count > 0)
+            {
+                var original = OriginalLines.Pop();
+                var line = original.Item1;
+                var content = original.Item2;
```

Comment:
Why do we assign these to variables never used again?

Reply:
Actually the whole member will be nuked.